jdaviz/configs/imviz/helper.py

```diff
@@ -105,6 +105,31 @@ def load_data(self, data, parser_reference=None, **kwargs):
             self.app.load_data(
                 data, parser_reference=parser_reference, **kwargs)
 
+    def link_data(self, link_type='pixels', wcs_fallback_scheme='pixels', error_on_fail=False):
```

Comment:
@astrofrog , this is what I have in mind for the "toggle". Does this make sense to you? I'd rather you tell me now before I implement the code.

Reply:
p.s. Updated the signature. Check the current diff.